system/security/plugins/htpasswdSecurity/htpasswdSecurity.cpp

```diff
@@ -60,14 +60,32 @@ class CHtpasswdSecurityManager : public CBaseSecurityManager
 		userMap.kill();
 	}
 
-	secManagerType querySecMgrType()
+    uint64_t queryFeatures(SecurityFeatureSupportLevel level, IEspSecureContext* secureContext = nullptr) const override
+    {
+        uint64_t mask = CBaseSecurityManager::queryFeatures(level, secureContext);
+        switch (level)
+        {
+        case SFSL_Safe:
+            mask = mask | s_safeFeatureMask;
+            break;
+        case SFSL_Implemented:
+            mask = mask | s_implementedFeatureMask;
+            break;
+        case SFSL_Unsafe:
+        default:
+            break;
+        }
+        return mask;
+    }
+
+    secManagerType querySecMgrType(IEspSecureContext* secureContext = nullptr)
 	{
 		return SMT_HTPasswd;
 	}
 
-	inline virtual const char* querySecMgrTypeName() { return "htpasswd"; }
+    inline virtual const char* querySecMgrTypeName(IEspSecureContext* secureContext = nullptr) { return "htpasswd"; }
```

Comment:
Formatting got all messed up in this source file. Not just here but in many places

Reply:
I probably copied and pasted changes from elsewhere before I realized how many tabs were in this file to start.